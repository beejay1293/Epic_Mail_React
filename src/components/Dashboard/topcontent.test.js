import React from 'react';
import { mount, shallow, render } from 'enzyme';
import Topcontent from './topcontent';
describe('Top content', () => {
  it('should render correctly topcontent', () => {
    const top = shallow(<Topcontent />);
  
    expect(top).toMatchSnapshot();
  });
});
