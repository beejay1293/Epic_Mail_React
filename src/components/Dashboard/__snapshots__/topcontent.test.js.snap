// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Top content should render correctly topcontent 1`] = `
<div
  className="top-content"
>
  <input
    className="first"
    type="checkbox"
  />
  <i
    className="fas fa-redo-alt reload"
    id="reload"
  />
</div>
`;
